src/ApiService/ApiService/OneFuzzTypes/Enums.cs

```diff
@@ -294,6 +294,7 @@ public enum AgentMode {
 }
 
 public enum NodeState {
+    New,
```

Comment:
I don't think we need the new state. 
The implementation of NodeOperations.SetState does not match the python code. 
- [in python](https://github.com/chkeita/onefuzz/blob/3347e7e67b3c51dddf47eafc9320cfbc7bd609ae/src/api-service/__app__/onefuzzlib/workers/nodes.py#L486) the state is saved even if the state is the same (we don't send the event in that case).
- [in C#](https://github.com/chkeita/onefuzz/blob/4662df3e397c6bb1406647515a47b4fa868c4f26/src/ApiService/ApiService/onefuzzlib/NodeOperations.cs#L450) we don't save the node if the state is the same.

Reply:
yeah you are right.

Reply:
New state could be good - since it makes SetState implementation consistent across all components.

Reply:
yes adding  new state might be beneficial. But i think we should add it after the migration is done. 
Keeping things the same  simplify the comparison with the behavior of the python code.